sql/cmd.go

```diff
@@ -24,8 +24,9 @@ func hasDatabaseConnector(driverName string) bool {
 		return tryRun("python", "-c", "from mysql.connector import connect")
 	} else if driverName == "sqlite3" {
 		return tryRun("python", "-c", "from sqlite3 import connect")
+	} else if driverName == "maxcompute" {
+		return true
```

Comment:
Maybe change this to `tryRun("python", "-c", "import pyodps")` and update the Dockerfile? This can be done in the next PR. :)

Reply:
Good idea!